### ISSUE TITLE
Connection to Raspi 1 with 26 Pin GPIO

### ISSUE THREAD

**Issue by burakozhan** [mathisschmieder/MMDVM_HS_Hat#14]
Hi There ;) 
Long time no see ...

tl dr;
Question about the board here; Is there an "official" way to connect the MMDVM_HS_pi_hat to an old Raspberry Pi Version 1 with the 26 pin GPIO header?

Detailed version;
In MMDVM HAT Versions sold on "chinese-wholesale-website" (aliexpress) and "formerly-marketplace-now-onlineshop" (eBay) there are 2 pcs of 2x5 headers, one on pins 1-10 and another on pins 31-40. According to the Schematic you only use 2 pins from the high numbered block. BOOT0 and NRST. These correspond to GPIO20 and GPIO21 of the Raspberry Pi. 
These pins were not available on the old Raspi1. (only starting with Raspi1 B+)
Since the old board has the same CPU as the Raspberry Pi 0, it's performance should be similar. Yet the old Raspi has the advantage of a wired Ethernet port, which I want to make use of. (Also it is sitting in a drawer collecting dust.)

I tried to;
connect only the first 10 pins (on a Raspi4 board where I know everything works when all pins are connected.) But that causes erratic behaviour. I.Ex. `pistar-findmodem` command only works once, after a power-cycle. Probably because the reset does not occur correctly.

Is there any specific reason why GPIO20 and GPIO21 were used, like besides mechanical stability or easier routing?

I guess I would need to re-compile the firmware and the Host software if I swap the pins to some other GPIO, is that assumption correct?
